forge-game/src/main/java/forge/game/player/actions/PlayerAction.java

```diff
@@ -0,0 +1,22 @@
+package forge.game.player.actions;
+
+import forge.game.GameEntityView;
+import forge.game.player.PlayerController;
+
+public abstract class PlayerAction {
+    protected String name;
+    protected GameEntityView gameEntityView = null;
+
+    public PlayerAction(GameEntityView cardView) {
+        gameEntityView = cardView;
+    }
+
+    public void run(PlayerController controller) {
```

Comment:
Do I still need this? Should we move the manager actions into these functions instead?

Reply:
This didn't work since they were in different modules